modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -641,6 +654,8 @@ public void testAddObjectEntry() throws Exception {
 					"\"listTypeEntryKeyRequired\"",
 				objectEntryValuesException.getMessage());
 		}
+
+		_testOldValue();
```

Comment:
@p-albuquerque I think this would be more appropriate inside the `testAddObjectEntryWithObjectValidationRule`

Reply:
In addition, we could rename it to something like `testAddObjectEntryWithObjectValidationOldValueRule`